Debian/8/input/oval/installed_OS_is_debian8.xml

```diff
@@ -10,21 +10,12 @@
       <reference source="JL" ref_id="DEBIAN8_20151210" ref_url="test_attestation" />
     </metadata>
     <criteria comment="current Debian version is Debian jessie" operator="AND">
-      <criterion comment="Installed operating system is part of the unix family" test_ref="test_unix_family" />
+      <extend_definition comment="Installed OS is part of the Unix family" definition_ref="installed_OS_is_part_of_Unix_family" />
```

Comment:
@iankko why add a separate OVAL file when the id attributes could be changed to something like `object_ref="obj_platform_unix_family"` like it is with `installed_OS_is_rhel7.xml`?